admin_requests.go

```diff
@@ -0,0 +1,41 @@
+package compute
+
+import (
+	"encoding/json"
+	"net/http"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// EditorToken aquires and sets the editor token for the provided Runnable struct.
+// See: Runnable.Token()
+func (c *Client) EditorToken(runnable *Runnable) (*http.Response, error) {
+	// GET /api/v1/token/{environment}.{customerID}/{namespace}/{fnName}
+	req, err := c.adminRequestBuilder(http.MethodGet, path.Join("/api/v1/token", runnable.Path()), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return res, err
+	}
+
+	token := TokenResponse{}
+	dec := json.NewDecoder(res.Body)
```

Comment:
Could we instead use JSON Unmarshal here?

Reply:
`json.Unmarshal` takes a `[]byte` slice, whereas `NewDecoder` takes an `io.Reader` interface, which is what `res.Body` is. I'm pretty sure doing it this way pipes the HTTP response output stream directly into the decoder and saves us a memcpy somewhere.

This is a pretty common pattern in Go.